libraries/core_libs/network/src/tarcap/packets_handlers/get_pbft_sync_packet_handler.cpp

```diff
@@ -32,38 +32,58 @@ void GetPbftSyncPacketHandler::process(const PacketData &packet_data,
   // Here need PBFT chain size, not synced period since synced blocks has not verified yet.
   const size_t my_chain_size = pbft_chain_->getPbftChainSize();
   size_t blocks_to_transfer = 0;
+  auto will_be_synced = false;
   if (my_chain_size >= height_to_sync) {
```

Comment:
I believe that any node, which is requesting period higher than our current period should be considered malicious because we provide info about our actual  chain size so other node should never request period higher than what we provided to it(please double check this), so here could be something like this:

```
if (my_chain_size < height_to_sync) {
    throw MaliciousPeerException("err message");
}
```

Reply:
I agree. Fixed, thanks